From 127a73a4e8297cecf0e81f029ca8e86f03df114b Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 12 Apr 2020 09:35:22 +0300
Subject: [PATCH 125/133] fix my derp v2

	modified:   core/res/res/values-night/colors.xml
	modified:   core/res/res/values-night/values.xml
	modified:   core/res/res/values/colors_device_defaults.xml
	modified:   packages/SystemUI/res/values-night/colors.xml
	modified:   packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
	modified:   packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
---
 core/res/res/values-night/colors.xml            |  2 +-
 core/res/res/values-night/values.xml            |  4 ++--
 core/res/res/values/colors_device_defaults.xml  |  4 ++--
 packages/SystemUI/res/values-night/colors.xml   |  2 +-
 .../res/values-night/colors.xml                 | 15 ++++++++-------
 .../res/values-night/colors.xml                 | 17 +++++++++--------
 6 files changed, 23 insertions(+), 21 deletions(-)

diff --git a/core/res/res/values-night/colors.xml b/core/res/res/values-night/colors.xml
index c5e72f012a2..5892455ddaf 100644
--- a/core/res/res/values-night/colors.xml
+++ b/core/res/res/values-night/colors.xml
@@ -27,7 +27,7 @@
     <color name="notification_default_color_dark">#ddffffff</color>
 
     <!-- The background color of a notification card. -->
-    <color name="notification_material_background_color">@color/black</color>
+    <color name="notification_material_background_color">@color/primary_dark_device_default_dark</color>
 
     <color name="chooser_row_divider">@color/list_divider_color_dark</color>
     <color name="chooser_gradient_background">@color/loading_gradient_background_color_dark</color>
diff --git a/core/res/res/values-night/values.xml b/core/res/res/values-night/values.xml
index 4e6b712f1f5..41b1a9c477f 100644
--- a/core/res/res/values-night/values.xml
+++ b/core/res/res/values-night/values.xml
@@ -27,7 +27,7 @@
         <item name="forceDarkAllowed">false</item>
 
         <!-- QS panel background -->
-        <item name="colorBackgroundFloating">@color/black</item>
+        <item name="colorBackgroundFloating">@color/primary_dark_device_default_dark</item>
 
         <!-- volume background -->
         <item name="panelColorBackground">@color/material_grey_800</item>
@@ -39,4 +39,4 @@
         <item name="textColor">@color/notification_secondary_text_color_dark</item>
         <item name="textSize">@dimen/notification_text_size</item>
     </style>
-</resources>
\ No newline at end of file
+</resources>
diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 80a1b9427b8..f120c738531 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -21,7 +21,7 @@
     <color name="primary_device_default_light">@color/primary_material_light</color>
     <color name="primary_device_default_settings">#ff2d2d2d</color>
     <color name="primary_device_default_settings_light">#ffffffff</color>
-    <color name="primary_dark_device_default_dark">#ff242424</color>
+    <color name="primary_dark_device_default_dark">#ff151618</color>
     <color name="primary_dark_device_default_light">@color/primary_dark_material_light</color>
     <color name="primary_dark_device_default_settings">#ff000000</color>
     <color name="primary_dark_device_default_settings_light">#ffffffff</color>
@@ -39,7 +39,7 @@
 
     <color name="background_device_default_dark">@color/background_material_dark</color>
     <color name="background_device_default_light">#ffffffff</color>
-    <color name="background_floating_device_default_dark">#ff3c4043</color>
+    <color name="background_floating_device_default_dark">@color/primary_dark_device_default_dark</color>
     <color name="background_floating_device_default_light">@color/background_floating_material_light</color>
 
     <!-- Error color -->
diff --git a/packages/SystemUI/res/values-night/colors.xml b/packages/SystemUI/res/values-night/colors.xml
index 7b55d18676d..9d95c796135 100644
--- a/packages/SystemUI/res/values-night/colors.xml
+++ b/packages/SystemUI/res/values-night/colors.xml
@@ -25,7 +25,7 @@
     <color name="notification_legacy_background_color">@*android:color/notification_material_background_color</color>
 
     <!-- The color of the material notification background when dimmed -->
-    <color name="notification_material_background_dimmed_color">#aa000000</color>
+    <color name="notification_material_background_dimmed_color">@*android:color/notification_material_background_color</color>
 
     <!-- The color of the dividing line between grouped notifications while . -->
     <color name="notification_divider_color">#212121</color>
diff --git a/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml b/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
index a523cee654d..e34aeb973dc 100644
--- a/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
+++ b/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
@@ -20,18 +20,19 @@
     <color name="notification_primary_text_color_dark">#ffe5e5e5</color>
     <color name="notification_secondary_text_color_dark">#b3e5e5e5</color>
     <color name="notification_default_color_dark">#ffe5e5e5</color>
-    <color name="notification_material_background_color">#ff0f1012</color> <!-- material_grey_900 -->
+    <color name="notification_material_background_color">#ff151618</color> <!-- material_grey_900 -->
     <color name="primary_material_dark">#ff0f1012</color> <!-- material_grey_900 -->
     <color name="primary_material_settings">#ff0f1012</color> <!-- material_grey_900 -->
     <color name="primary_dark_material_dark">@android:color/black</color>
-    <color name="primary_dark_material_settings">#ff0f1012</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_dark">#ff0f1012</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_settings">#ff0f1012</color> <!-- material_grey_850 -->
+    <color name="primary_dark_material_settings">#ff151618</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_dark">#ff151618</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_settings">#ff0f1012</color> <!-- main settings bg -->
+    <color name="background_floating_device_default_dark">#ff151618</color>
     <color name="secondary_material_settings">#ff1a1b1e</color> <!-- material_grey_800 -->
-    <color name="tertiary_material_settings">#ff202126</color>
+    <color name="tertiary_material_settings">#ff202126</color> <!-- material_grey_600 -->
     <color name="quaternary_material_settings">#ffdadce0</color> <!-- material_grey_200 -->
-    <color name="material_grey_900">#ff0f1012</color>
-    <color name="material_grey_850">#ff0f1012</color>
+    <color name="material_grey_900">#ff151618</color>
+    <color name="material_grey_850">#ff151618</color>
     <color name="material_grey_800">#ff1a1b1e</color>
     <color name="material_grey_600">#ff202126</color>
     <color name="material_grey_300">#ff3c4043</color>
diff --git a/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml b/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
index de50bd007f8..42de11c6e6c 100644
--- a/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
+++ b/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
@@ -20,18 +20,19 @@
     <color name="notification_primary_text_color_dark">#ffe7faff</color>
     <color name="notification_secondary_text_color_dark">#b3e7faff</color>
     <color name="notification_default_color_dark">#ffe7faff</color>
-    <color name="notification_material_background_color">#ff0d2029</color> <!-- material_grey_900 -->
+    <color name="notification_material_background_color">#ff182a33</color> <!-- material_grey_900 -->
     <color name="primary_material_dark">#ff0d2029</color> <!-- material_grey_900 -->
     <color name="primary_material_settings">#ff0d2029</color> <!-- material_grey_900 -->
     <color name="primary_dark_material_dark">#ff09181e</color>
-    <color name="primary_dark_material_settings">#ff0d2029</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_dark">#ff0d2029</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_settings">#ff0d2029</color> <!-- material_grey_850 -->
+    <color name="primary_dark_material_settings">#ff182a33</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_dark">#ff182a33</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_settings">#ff0d2029</color> <!-- main settings bg -->
+    <color name="background_floating_device_default_dark">#ff182a33</color>
     <color name="secondary_material_settings">#ff1b2e38</color> <!-- material_grey_800 -->
-    <color name="tertiary_material_settings">#ff1e343f</color>
-    <color name="quaternary_material_settings">#ff3c4043</color> <!-- material_grey_200 -->
-    <color name="material_grey_900">#ff0d2029</color>
-    <color name="material_grey_850">#ff0d2029</color>
+    <color name="tertiary_material_settings">#ff1e343f</color> <!-- material_grey_600 -->
+    <color name="quaternary_material_settings">#ff9aadb7</color> <!-- material_grey_200 -->
+    <color name="material_grey_900">#ff182a33</color>
+    <color name="material_grey_850">#ff182a33</color>
     <color name="material_grey_800">#ff1b2e38</color>
     <color name="material_grey_600">#ff1e343f</color>
     <color name="material_grey_300">#ff9db3be</color>
-- 
2.17.1

